### ISSUE TITLE
README possible typo in configuration example

### ISSUE THREAD

**Issue by richbrill** [themikenicholson/passport-jwt#106]
In the configuration example:

```
passport.use(new JwtStrategy(opts, function(jwt_payload, done) {
    User.findOne({id: jwt_payload.sub}, function(err, user) {
        if (err) {
            return done(err, false);
        }
        if (user) {
            done(null, user);
        } else {
            done(null, false);
            // or you could create a new account
        }
    });
}));
```

of the README.md file should the second and third done's actually be returned, and not just invoked?

**Comment by themikenicholson** [comment]
Ah, you're right.  Thanks. 

Fixed in c135b28f63adcbde76b7c57e61cf96640af6e362

**Comment by richbrill** [comment]
No probs
